Source/IntegrationTests/TestFiles/LitTests/LitTest/verification/nonLinearArithmetic.dfy

```diff
@@ -0,0 +1,104 @@
+// RUN: ! %verify "%s" --disable-nonlinear-arithmetic --resource-limit 1000 > "%t"
```

Comment:
I would put two RUN here, one with --disable-non-linear-arithmetic and one without. That way, we can verify that the output of Power1 no longer depends on this flag.

Reply:
Good idea